modules/usage/module.php

```diff
@@ -564,22 +578,29 @@ private function save_document_usage( Document $document ) {
 		// Get data manually to avoid conflict with `\Elementor\Core\Base\Document::get_elements_data... convert_to_elementor`.
 		$data = $document->get_json_meta( '_elementor_data' );
 
-		if ( ! empty( $data ) ) {
-			try {
-				$usage = $this->get_elements_usage( $document->get_elements_raw_data( $data ) );
+		try {
+			$this->add_document_usage( $document, $data );
+		} catch ( \Exception $exception ) {
+			return; // Do nothing.
```

Comment:
Why not log it?
Plugin::$instance->logger->get_logger()->error( $exception->getMessage(), [
					'document_id' => $document->get_id(),
					'document_name' => $document->get_name(),
				] );

Reply:
Handled.